src/main/java/core/basesyntax/services/DataConverterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.services;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        return data.stream().skip(1)

```

Comment:
Skipping the first line is a hardcoded behavior assuming the first line is always a header. This might not be flexible for different kinds of CSV files. Consider passing a parameter to the method specifying whether to skip the first line or not, or even better, make the decision based on the actual content of the first line.